src/main/java/com/netflix/imflibrary/st2067_2/CompositionImageEssenceDescriptorModel.java

```diff
@@ -388,6 +400,217 @@ public TransferCharacteristic getTransferCharacteristic() {
         return paletteLayout;
     }
 
+    static public class J2KHeaderParameters {
```

Comment:
This isn't a major problem,  but how practical would it be to reuse and enhance the _existing_ [JPEG2000PictureSubDescriptor](https://github.com/Netflix/photon/blob/beda2484b5df63b0383a463841eaa8da6c5d14f7/src/main/java/com/netflix/imflibrary/st0377/header/JPEG2000PictureSubDescriptor.java) class, rather than defining a new `J2KHeaderParameters`? My understanding, is that both classes essentially represent the same information, right?

We have similar cases all throughout Photon. Usually, one class will be defined once for a piece of data extracted from the MXF file, and then a _second_ class will be defined elsewhere, to represent the same piece of data, when it is extracted from the CPL. Ideally, both cases should be sharing the same Java class.

This is an area of tech debt that we would like to clean up in a future revision of Photon.
Obviously, all of those other duplicate classes are unrelated to your pull request. And you don't _need_ to change anything right now, especially if that is not a practical change to make. 
But it might be worth considering. Changing it now is much easier than reorganizing things in a future refactoring, once people begin to reference `J2KHeaderParameters`.

Reply:
> This isn't a major problem, but how practical would it be to reuse and enhance the existing JPEG2000PictureSubDescriptor class, rather than defining a new J2KHeaderParameters?

Ok. Will give a shot.